File: CSTH/NEW/test2/pack1/Adjacent.py
class Adjacent:
	def __init__(self) -> None:
		self.tapDinhKe = []
	
	def addDinhKe(self, u):
		self.tapDinhKe.append(u)
	
	def deleteDinhKe(self, u):
		self.tapDinhKe.remove(u)

File: CSTH/OLD/test2/pack1/Adjacent.java
package pack1;
import java.util.*;

public class Adjacent{
  public List<Integer> tapDinhKe;
  public Adjacent() {
    tapDinhKe = new ArrayList<Integer>();
  }
  public void addDinhKe(int u) {
    tapDinhKe.add(u);
  }
  public void deleteDinhKe(int u) {
    tapDinhKe.remove((Integer) u);
  }
}